static/app/views/organizationGroupDetails/grouping/grouping.tsx

```diff
@@ -1,139 +1,304 @@
+                {t('Unmerge issue')}
+              </Button>
+            }
+          >
+            {error.responseJSON.detail}
+          </EmptyMessage>
+        </Wrapper>
+      );
+    }
+
     return (
       <LoadingError
-        message={t('Unable to load grouping data, please try again later')}
-        onRetry={fetchData}
+        message={t('Unable to load grouping levels, please try again later')}
+        onRetry={fetchGroupingLevels}
       />
     );
   }
 
-  const mergedIssuesWithLastestEvent =
-    unMergedIssueIds.length > 0
-      ? mergedIssues.filter(
-          ({id, latestEvent}) => !unMergedIssueIds.includes(id) && !!latestEvent
-        )
-      : mergedIssues.filter(({latestEvent}) => !!latestEvent);
+  if (!groupingLevels.length) {
+    return (
+      <EmptyStateWarning withIcon={false}>
+        {t('No grouping levels have been found.')}
+      </EmptyStateWarning>
+    );
+  }
+
+  //function handleRegroup() {
+  // Todo(Priscila): Implement it
+  //}
 
   return (
     <Wrapper>
-      <MergedIssues
-        groupId={groupId}
-        project={project}
-        organization={organization}
-        location={location}
-        issues={mergedIssuesWithLastestEvent}
-        selectedIssueIds={selectedIssueIds}
-        pagination={mergedIssuesPagination}
-        enableCompareButton={enableCompareButton}
-      />
-      <GroupedIssues issues={mergedIssuesWithLastestEvent} groupId={groupId} />
+      <Description>
+        {t(
+          'Sometimes you might want to split up the errors in an issue by different frames in the stacktrace. Below you can select which frames to regroup this issue by and see how many new issues will be created in the process.'
+        )}
+      </Description>
+      <div>
+        <StyledList symbol="colored-numeric">
+          <StyledListItem>
+            {t('Select levels')}
+            <StyledRangeSlider
+              name="grouping-level"
+              allowedValues={groupingLevels.map(groupingLevel =>
+                Number(groupingLevel.id)
+              )}
+              formatLabel={value => {
+                return value === 0
+                  ? t('Automatically grouped')
+                  : tn('%s level', '%s levels', value);
+              }}
+              value={activeGroupingLevel ?? 0}
+              onChange={groupingLevelId =>
+                setActiveGroupingLevel(Number(groupingLevelId))
+              }
+            />
+          </StyledListItem>
+          <StyledListItem>
+            {isGroupingLevelDetailsLoading ? (
+              <div>
+                <div>{t('What happens to this issue')}</div>
+                <LoadingIndicator mini />
+              </div>
+            ) : (
+              <Fragment>
+                <div>
+                  {t('What happens to this issue')}
+                  <WhatHappensDescription>
+                    {tct(
+                      `This issue will be deleted and [quantity] new issues will be created.`,
```

Comment:
```suggestion
                      `This issue will be deleted and [quantity]+ new issues will be created.`,
```

Not sure if there's a better solution at the moment, I have created an asana task for providing you with a better API for this number.